lib/kubernetes-deploy/kubernetes_resource/resource_quota.rb

```diff
@@ -0,0 +1,33 @@
+# frozen_string_literal: true
+module KubernetesDeploy
+  class ResourceQuota < KubernetesResource
+    TIMEOUT = 30.seconds
+
+    def sync
+      raw_json, _err, st = kubectl.run("get", type, @name, "--output=json")
+      @status = st.success? ? "Available" : "Unknown"
+      @found = st.success?
+      @rollout_data = if @found
+        JSON.parse(raw_json)
+      else
+        {}
+      end
+    end
+
+    def deploy_succeeded?
+      @rollout_data["spec"]["hard"] == @rollout_data["status"]["hard"]
```

Comment:
You'll need to `dig` here since `@rollout_data` can be an empty hash

Reply:
Fixed :)